docs/pipelineruns.md

```diff
@@ -15,7 +15,7 @@ Creation of a `PipelineRun` will trigger the creation of
   - [Resources](#resources)
   - [Service account](#service-account)
 - [Cancelling a PipelineRun](#cancelling-a-pipelinerun)
-- [Examples](#examples)
+- [Examples](https://github.com/tektoncd/pipeline/tree/master/examples/pipelineruns)
```

Comment:
I think you can make this relative to the root of the repo (and make sure the link works in PRs etc.) like this:

` [Examples](examples/pipelineruns)`

Reply:
The relative one not work 😭 , I tried it in my fork

Reply:
whoops sorry my bad! this in the docs dir so it would actually be `[Examples](../examples/pipelineruns)`!! :pray::pray::pray:

(you can see that same syntax in action in https://github.com/tektoncd/pipeline/blob/master/docs/install.md#custom-releases)

Reply:
just bumping this if you have a chance to make the change @vincent-pli - if not we can merge as is, but having a relative link makes the link work better in the context of a PR if possible 🙏